dashboard/src/shared/Config.test.ts

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import Config from "./Config";
+
+describe("Config", () => {
+  let defaultJSON: any;
+  let initialEnv: string;
+
+  beforeAll(() => {
+    initialEnv = process.env.REACT_APP_KUBEAPPS_NS || "";
+    const mock = new MockAdapter(axios);
+
+    defaultJSON = require("../../public/config.json");
+
+    mock.onGet("/config.json").reply(200, defaultJSON);
+  });
+
+  afterEach(() => {
```

Comment:
should this be beforeEach?

Reply:
No.

The idea of the afterEach is to reset any env variable change has been produced in this test context to avoid side-effects in other test files.